operations/http.go

```diff
@@ -291,11 +292,13 @@ func (ac *legacyClient) GetPatchedConfig(patchId string) (*model.Project, error)
 		return nil, NewAPIError(resp)
 	}
 	ref := &model.Project{}
+	ctx := context.Background()
```

Comment:
Can pass this context in from the ones defined in the caller

Reply:
done